src/self-help/README.md

```diff
@@ -0,0 +1,5 @@
+# Azure CLI SelfHelp Extension #
+This is an extension to Azure CLI to manage SelfHelp resources.
+
+## How to use ##
```

Comment:
Please add examples and useage of the commands in the extension

Reply:
Added examples and usage in readme.